lib/absinthe/subscription/redix_store.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Absinthe.Subscription.RedixStore do
```

Comment:
As a minor note, while I definitely appreciate this as an example, I definitely don't want to have Absinthe ship with this built in. It couples Absinthe too closely to a specific redis library. I'd be more than happy however to promote an external package that contained this code however.

Reply:
That's fine, as I said, it's only an illustration.